config/index.js

```diff
@@ -16,7 +16,7 @@ const config = {
         },
         logger: {
             silent: false,
-            level: process.env.LOGLEVEL || 'debug'
+            level: process.env.LOGLEVEL || 'info'
```

Comment:
better to override it at production config and leave the default as debug, we only need info in production